openedx/features/smart_referral/models.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class SmartReferral(TimeStampedModel):
```

Comment:
Lets add `__unicode__()` and return something like `{username} reffered {contact_email}`